### ISSUE TITLE
Increased Mid-Arm Muscle Area

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5899]
**Preferred term label:** Increased Mid-Arm Muscle Area

**Synonyms** Increased MAMA

**Definition (free text, please give PubMed ID)** Estimation of the area of the mid-upper arm that is composed of muscle is increased., PMID: 30558577

**Parent term (use hpo.jax.org/app)** Abnormality of the upper arm HP:0001454

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
PMID:27806975

added term
